webapp/lib/app/configurator/standard_messages/controller.dart

```diff
@@ -117,41 +119,51 @@ class MessagesConfiguratorController extends ConfiguratorController {
             standardMessage.groupDescription: [standardMessage]
           }
         });
+        _updateUnsavedIndicators(standardMessagesManager.categories, standardMessagesManager.unsavedMessageIds, standardMessagesManager.unsavedGroupIds, standardMessagesManager.unsavedCategoryIds);
         break;
 
       case MessagesConfigAction.addStandardMessagesGroup:
         StandardMessagesGroupData groupData = data;
         var newGroup = standardMessagesManager.createStandardMessagesGroup(groupData.category);
         _addMessagesToView({groupData.category: {newGroup.groupDescription: []}}, startEditingName: true);
+        _updateUnsavedIndicators(standardMessagesManager.categories, standardMessagesManager.unsavedMessageIds, standardMessagesManager.unsavedGroupIds, standardMessagesManager.unsavedCategoryIds);
         break;
 
       case MessagesConfigAction.updateStandardMessagesGroup:
         StandardMessagesGroupData groupData = data;
         standardMessagesManager.renameStandardMessageGroup(groupData.category, groupData.group, groupData.newGroupName);
         _view.categoriesByName[groupData.category].renameGroup(groupData.group, groupData.newGroupName);
+        _view.categoriesByName[groupData.category].markAsUnsaved(true);
+        _view.categoriesByName[groupData.category].groupsByName[groupData.newGroupName].markAsUnsaved(true);
```

Comment:
do we still need these `markAsUnsaved(true)` if we call `_updateUnsavedIndicators` right after?